### PR TITLE
Fix coveralls

### PR DESCRIPTION
**Pull request by higs4281**
Add a run_travis script and travis matrix vars

This produces separate parallel builds for front and back ends and should
fix our blank coverage badge. ~~but the coverage report for js still doesn't pick up data~~

## Additions

- run_travis.sh to launch separate builds

## Removals

- the `coverage combine` command, which wasn't doing the job

## Review

- @chosak @contolini 

## Testing

Check out the jobs at https://coveralls.io/builds/8022169

selfie-0
[_GitHub Selfies_](https://github.com/thieman/github-selfies/)

## Todos
- ~~I could use some front-end pairing to work on js reporting~~ -- done

## Checklist

* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by higs4281** [comment]
@chosak got both code bases reporting to coveralls.

**Comment by contolini** [comment]
Siiiiiiiick!

File: run_travis.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+echo "running $RUNTEST"
+if [ "$RUNTEST" == "frontend" ]; then
+    gulp lint
```
Comment (sebworks) [inline run_travis.sh]: not sure you need to run the linter here

Comment (higs4281) [inline run_travis.sh]: sold.

**Comment by sebworks** [comment]
👍

File: run_travis.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+echo "running $RUNTEST"
```
Comment (sebworks) [inline run_travis.sh]: should this be `running coveralls $RUNTEST`?

**Pull request merged**